requirements.txt

```
@@ -1,4 +1,4 @@
-sphinx==3.3.0
+sphinx==4.0.1
```

Let's use the latest bugfix release, no need to hold back:
```suggestion
sphinx==4.0.3
```

---

@webknjaz is it the best choice to use v4.0.3? I think it is still in development.